lib/src/settings.rs

```diff
@@ -88,6 +88,10 @@ impl UserSettings {
         }
     }
 
+    pub fn open_commits(&self) -> bool {
+        self.config.get_bool("ui.open-commits").unwrap_or(true)
```

Comment:
Nit: ambiguity here between whether this is "open commits" as a noun or as a verb. Not sure if that matters too much, but if it's not too difficult, you may want to name this 'ui.enable-open-commits' or 'experimental.enable-open-commits' or something?

Reply:
Good point. I've renamed it.